libraries/resource_systemd_automount.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'resource_systemd_unit'
+require_relative 'systemd_automount'
+
+class Chef::Resource
+  class SystemdAutomount < Chef::Resource::SystemdUnit
+    self.resource_name = :systemd_automount
```

Comment:
so as it turns out if you don't, rubocop'll complain about the useless variable assignment, so i went ahead and left it as is, since it otherwise doesn't seem to matter.